pkg/asset/installconfig/ovirt/credentials.go

```diff
@@ -2,12 +2,25 @@ package ovirt
 
 import (
 	"fmt"
+	"net/http"
 	"net/url"
 
 	"github.com/sirupsen/logrus"
 	"gopkg.in/AlecAivazis/survey.v1"
 )
 
+// Check if URL can be reached before we proceed with the installation
+// Parms:
+//	urlAddr - Full URL
```

Comment:
this is inconsistent with functional comments, for Go we prefer explaining the inputs in description of the function instead of a separate section for input. like `checkURLResponse performs a GET on the provided urlAddr to ensure that the url actually exists`

also this function name doesn't explain what check is being performed on the urlAddr.
also urlAddr is just redundant in naming, `url` should be enough.

secondly what about https URLs, do we need to skip trusted check?

Reply:
@abhinavdahiya I will prepare a new PR. Thanks for the comments.